pkg/metrics/node-cleanup/metrics.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	// LocalVolumeNodeCleanupSubsystem is prometheus subsystem name.
+	LocalVolumeNodeCleanupSubsystem = "local_volume_node_cleanup"
+)
+
+var (
+	// APIServerRequestsTotal is used to collect accumulated count of apiserver requests.
+	APIServerRequestsTotal = prometheus.NewCounterVec(
```

Comment:
Done.

Reply:
If you important client-go metrics I think you can remove this one:
https://github.com/kubernetes/kubernetes/blob/99190634ab252604a4496882912ac328542d649d/cmd/kube-proxy/proxy.go#L24

Reply:
Sorry, where at? I don't think I have that import, just the import of the workqueue. And the client-go metrics should exist because they are in the default registry https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/5f99ede0e0899feecbf7fb6b348f53e05f5a72d9/cmd/node-cleanup/main.go#L102

Reply:
Done.